.github/workflows/build_and_test.yml

```diff
@@ -490,7 +495,7 @@ jobs:
           - os: windows-2022
             platform: "Visual Studio 17 2022"
             arch: "x64"
-            pcap_lib: "npcap"
+            pcap_lib: "winpcap"
```

Comment:
This is duplicate, maybe one of them is `Win32`

https://github.com/seladb/PcapPlusPlus/blob/1fd0df5510d237fdb7db639e2559ac2cdb9b87e6/.github/workflows/build_and_test.yml#L491-L498

Reply:
it's all just temporary because the Npcap web-site and its resources are all down for at least a day or two 😕 
Is https://npcap.com/ working for you? 🤔 
I wanted to test that my code works, and it's not related to WinPcap/Npcap in any way, so I figured I'll use WinPcap until Npcap is back...

Reply:
I understand that this is a temporary change but original matrix include 1 Win32 job and 3 x64 job. I mean since we have two options (npcap and winpcap), two versions (2019 and 2022) and two archs (win32 and x64) there should be more jobs 😁

Reply:
More clearly, did we forget to add or we tried to decrease testing time/workflow count?

Reply:
we can have more options, but I think the most common options today are the combination of x64, Npcap and vs2019/2022. Win32 and WinPcap are mostly deprecated I believe